adapter/src/main/java/com/twilio/kudu/sql/KuduEnumerable.java

```diff
@@ -518,11 +519,14 @@ public Schema getTableSchema() {
     List<AsyncKuduScanner> scanners = predicates.stream().map(subScan -> {
       KuduScanToken.KuduScanTokenBuilder tokenBuilder = client.syncClient()
           .newScanTokenBuilder(calciteKuduTable.getKuduTable());
-      if (sort) {
-        // Allows for consistent row order in reads as it puts in ORDERED by Pk when
-        // faultTolerant is set to true
-        tokenBuilder.setFaultTolerant(true);
-      }
+      // Allows for consistent row order in reads as it puts in ORDERED by Pk when
+      // faultTolerant is set to true
+      // setFaultTolerant to true sets the ReadMode to READ_AT_SNAPSHOT
+      // setting snapshot time to 1ms before currentTimestamp to avoid latency issues.
```

Comment:
nit: "Snapshot time to 1 second before currentTimestamp"

Reply:
Its 1ms as the tokenbuilder method is `snapshotTimestampMicros`. Are you suggesting to use 1second instead?

Reply:
Got it... Yes, can you try 1 second instead of 1ms.

Reply:
The IT tests fail if the snapshot time is set to 1s , so setting it back to 1ms. 1s delay will also cause the cluster tests to fail.